### ISSUE TITLE
[a11y-defect-2]: Payment History tables use display text that should be headings instead of paragraphs

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#95135]
### Point of contact

Trevor Pierce

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

We have several Bitter "headings" that are <p> tags styled to look like headings. This is a violation of SC 1.3.1 Info and Relationships. Screen reader uses often navigate by headings, so we want these blocks to appear in the Headings menu and respond to the h shortcut key. Screenshots attached below.

---

![Image](https://github.com/user-attachments/assets/77e87d28-433b-435e-948b-f2bc70d5d125)

### Reproduction steps

1. Log in with a test user who has a payment history with VA
2. Navigate to va-payment-history/payments/
3. Turn on the screen reader of your choice
4. Verify the Bitter sub-headings are not announced as headings

### Proposed solution or next steps

Update the Bitter font text "Displaying 1 - 6 of 87" to be an H3 and consider it saying "Displaying 1 - 6 of 87 payments" so it's clear what the proceeding data will be.

### References, articles, or WCAG support

1. Understanding [SC 1.3.1](https://w3.org/TR/WCAG22#info-and-relationships): Info and Relationships (Level A)
2. [Heading off confusion: When do headings fail WCAG?](https://www.tpgi.com/heading-off-confusion-when-do-headings-fail-wcag/)

### Type of issue

- [ ] Axe-core
- [ ] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [x] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [x] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else

**Comment by dysbo** [comment]
Refinement notes:
- Need to discuss with content folks (@jstrothman, @rallencf, others?) about making the text change outlined above

**Comment by rallencf** [comment]
I like @1Copenut's proposed solution, including using an <h3> tag. **Minor copyedit:** remove the space before and after the hyphen, per the [Content Style Guide](https://design.va.gov/content-style-guide/punctuation#em-and-en-dashes-hyphens). So, in the above example, the content should be:

<h3>Displaying 1-6 of 87 payments</h3>

@jstrothman Thoughts?

**Comment by rallencf** [comment]
@1Copenut I notice the table in the screenshots above uses `<span>` tags. Should we be using `<th>` and `<td>` tags for semantic markup?

**Comment by jstrothman** [comment]
Looks good to me!

**Comment by rallencf** [comment]
@janechodance Hi! Julie gave her thumbs up, so I reassigned the ticket to you to implement.
